spec/integ/crypto/verification.spec.ts

```diff
@@ -808,6 +810,115 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("verification (%s)", (backend: st
         });
     });
 
+    describe("Send verification request in DM", () => {
+        const crossSingingKeys: Partial<IDownloadKeyResult> = {
+            master_keys: {
+                "@bob:xyz": {
+                    keys: {
+                        "ed25519:i2y3s50NUzxxtIdyxEc9yFybLIq6qgBSb9r7k9yICBY":
+                            "i2y3s50NUzxxtIdyxEc9yFybLIq6qgBSb9r7k9yICBY",
+                    },
+                    user_id: "@bob:xyz",
+                    usage: ["master"],
+                },
+            },
+            self_signing_keys: {
+                "@bob:xyz": {
+                    keys: {
+                        "ed25519:cyTNi7Z7LZHlV1qnirzRypdsIWNLAH/z7Jc9PAIkf1g":
+                            "cyTNi7Z7LZHlV1qnirzRypdsIWNLAH/z7Jc9PAIkf1g",
+                    },
+                    user_id: "@bob:xyz",
+                    usage: ["self_signing"],
+                    signatures: {
+                        "@bob:xyz": {
+                            "ed25519:i2y3s50NUzxxtIdyxEc9yFybLIq6qgBSb9r7k9yICBY":
+                                "SrAMqvGSkwzxXoLZAcmuMl+gbQBlHbWObY1IX1wZ9ADlOJkeEzzjxzsyah25I29TLFHrTG+dFphreItE3pGQAA",
+                        },
+                    },
+                },
+            },
+            user_signing_keys: {
+                "@bob:xyz": {
+                    keys: {
+                        "ed25519:mdF9bCRhssc1OTWqsmSdS4usif2PboaT/cAJMNqykbA":
+                            "mdF9bCRhssc1OTWqsmSdS4usif2PboaT/cAJMNqykbA",
+                    },
+                    user_id: "@bob:xyz",
+                    usage: ["user_signing"],
+                    signatures: {
+                        "@bob:xyz": {
+                            "ed25519:i2y3s50NUzxxtIdyxEc9yFybLIq6qgBSb9r7k9yICBY":
+                                "7DoXVLc4iNvQQOSR9G9QQEot93M7CM76z0fUtraIo/hnMH1/M96DIskLl9MBONOL5nEekw2LXXFWh55AANMMCg",
+                        },
+                    },
+                },
+            },
+        };
+
+        const deviceKeys: IDeviceKeys = {
+            algorithms: ["m.olm.v1.curve25519-aes-sha2", "m.megolm.v1.aes-sha2"],
+            device_id: "bob_device",
+            keys: {
+                "curve25519:bob_device": "F4uCNNlcbRvc7CfBz95ZGWBvY1ALniG1J8+6rhVoKS0",
+                "ed25519:bob_device": "PNsWknI+bXfeeV7GFfnuBX3sATD+/riD0gvvv8Ynd5s",
+            },
+            user_id: "@bob:xyz",
+            signatures: {
+                "@bob:xyz": {
+                    "ed25519:bob_device":
+                        "T+UUApQlgZHNfM4gdkcrJcjTmeF7wEYAZH2OS99gKKizEgEFL89acAC7yuLfm2MI5J9Ks2AGkQJp3jjNjC/kBA",
+                },
+            },
+        };
```

Comment:
Hrm. can we use the `test_data` instead of this?

Reply:
`test_data` is about `alice` and I need keys for `bob` here. (I tested to just replace bob's username and device by Alice's username and device but it doesn't work)